classes/class3/homework_readable/routes/index.js

```diff
@@ -0,0 +1,68 @@
+var mongoose = require('mongoose');
+var names = require('./names');
+var colors = require('./colors');
+
+var cat = mongoose.Schema({
+	cat_name: String,
+	fur: String,
+	age: Number
+})
+
+var Cat = mongoose.model('Cat', cat);
+
+var make_cat = function(req,res){
+	var new_cat = new Cat;
+	new_cat.age = Math.round(Math.random()*20);
+	new_cat.cat_name = names.choose();
+	new_cat.fur = colors.choose();
+	new_cat.save();
+	res.render('add', {cat:new_cat});
+}
+
+var list_cats = function(req,res){
+	Cat.find().sort('-age').exec(function(err,cats){
+		if(err){
+			console.log(err);
+		}
+
+		else{
+			console.log(cats);
+			res.render('list',{catlist:cats});
+		}
+	});
+}
+
+var by_color = function(req,res,des_color){
```

Comment:
Somewhere along the line here you aren't getting the right value into `des_color`, because `/cats/bycolor/blue` (green, teal, etc.) never shows a single cat.